### ISSUE TITLE
olapR links are broken

### ISSUE THREAD

**Issue by minemR** [MicrosoftDocs/machine-learning-server-docs#91]
olapR links are broken:
https://github.com/MicrosoftDocs/machine-learning-server-docs/blob/live/sql/machine-learning/r/ref-r-olapr

where can I find olapR package code?

**Comment by s-polly** [comment]
@minemR 

The package is included in SQL Server Machine Learning Services and SQL Server 2016 R Services.  Please see this document for a location depending on your version: [olapR (R package in SQL Server Machine Learning Services)](https://learn.microsoft.com/en-us/sql/machine-learning/r/ref-r-olapr?view=sql-server-ver16#availability-and-location)

**Comment by minemR** [comment]
@s-polly Can olapR be used without installing SQL Server Machine Learning Services or SQL Server 2016 R Services?

**Comment by s-polly** [comment]
@minemR  I don't believe it can. The documentation I can find doesn't indicate as such. To get faster help I suggest you go to [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/) and see if you can find an answer there.

**Comment by s-polly** [comment]
#please-close
